.travis.yml

```diff
@@ -3,7 +3,6 @@ matrix:
   allow_failures:
     - rvm: ruby-head
 rvm:
-  - 2.0.0
   - 2.1.0
   - 2.2.5
   - 2.3.1
```

Comment:
What do you think about using the latest releases?
```
- 2.1.10
  - 2.2.9
  - 2.3.6
  - 2.4.3
  - 2.5.0
  - ruby-head
```

Reply:
Sounds good to me!